src/studio/src/designer/backend/Services/Implementation/RepositorySI.cs

```diff
@@ -93,16 +93,14 @@ public bool CreateServiceMetadata(ModelMetadata serviceMetadata)
                 ? Environment.GetEnvironmentVariable("ServiceRepositorySettings__RepositoryLocation") + serviceMetadata.Org.AsFileName()
                 : _settings.GetOrgPath(serviceMetadata.Org.AsFileName(), developerUserName);
 
-            string appPath = orgPath + "/" + serviceMetadata.RepositoryName.AsFileName();
+            string appPath = $"{orgPath}/{serviceMetadata.RepositoryName.AsFileName()}";
```

Comment:
A few lines up the orgPath is declared as null.
It could be directly assigned on line 92.

Reply:
I can fix it. I haven't really done changes in this file except a few minor things. This is the existing beast I would like to refactor away from, so I prefer to leave it more or less as is until I have replacements in place.